recaf-assembler/src/main/java/me/coley/recaf/assemble/suggestions/Suggestions.java

```diff
@@ -0,0 +1,184 @@
+package me.coley.recaf.assemble.suggestions;
+
+import me.coley.recaf.assemble.ast.arch.MethodDefinition;
+import me.coley.recaf.assemble.ast.arch.MethodParameter;
+import me.coley.recaf.assemble.util.ProgramClass;
+import me.coley.recaf.assemble.util.ProgramField;
+import me.coley.recaf.assemble.util.ProgramMethod;
+import me.coley.recaf.util.AccessFlag;
+import me.coley.recaf.util.ClasspathUtil;
+import me.darknet.assembler.instructions.ParseInfo;
+import me.darknet.assembler.parser.Group;
+import me.darknet.assembler.parser.groups.InstructionGroup;
+import org.objectweb.asm.Opcodes;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import static me.coley.recaf.util.ClasspathUtil.Tree;
+
+/**
+ * Suggestion text provider.
+ *
+ * @author Nowilltolife
+ * @author xDark
+ */
+public class Suggestions {
+	private final static SuggestionResult EMPTY = new SuggestionResult("", Stream.empty());
```

Comment:
Will cause exception

Reply:
Just moved it over from the old class. What is the intent here? I can update it.

Reply:
Need to create new result each time because Stream.empty() is not a singleton, see how it works